src/util/string_constant.cpp

```diff
@@ -12,25 +12,40 @@ Author: Daniel Kroening, kroening@kroening.com
 #include "c_types.h"
 #include "std_expr.h"
 
+static array_typet make_type(const irep_idt &value)
+{
+  exprt size_expr = from_integer(value.size() + 1, c_index_type());
```

Comment:
I am quite confused by the Codecov warnings here: isn't this code path necessarily hit if we ever construct a `string_constantt` for there is just one constructor? That would suggest that none of our tests ever result in constructing a `string_constantt`?!

Reply:
Seems more likely that the coverage job isn't working correctly. There are a couple of things I can think of to try -

- Removing `-Dparallel_tests=2`. Concurrent accesses to the coverage data files could be clobbering each other.
- There is a hint that building with `-fprofile-arcs` may be needed here -  [https://discourse.cmake.org/t/guideline-for-code-coverage/167](https://discourse.cmake.org/t/guideline-for-code-coverage/167)